elasticsearch.md

```diff
@@ -1369,21 +1394,25 @@ The `groups` rule accepts a list of group names. This rule will match if the res
 
     users:
 
-    - username: alice
-      auth_key: alice:p455phrase
+    - username: "alice"
       groups: ["team1"]
+      auth_key: alice:p455phrase
 
-    - username: bob
-      auth_key: bob:s3cr37
+    - username: "bob"
       groups: ["team2", "team4"]
+      auth_key: bob:s3cr37
 
-    - username: claire
-      auth_key_sha256: e0bba5fda92dbb0570fd2e729a3c8ed6b1d52b380581f32427a38e396ba28ec6 #claire:p455key
+    - username: "claire"
       groups: ["team1", "team5"]
+      auth_key_sha256: e0bba5fda92dbb0570fd2e729a3c8ed6b1d52b380581f32427a38e396ba28ec6 #claire:p455key
 ```
 
 _Example: rules are associated to groups \(instead of users\) and users-group association is declared separately later under `users:`_
 
+### External groups
```

Comment:
let's call the paragraph group mapping, so it appears as such in the ToC. This is a canned expression that people really scan for.